requirements.txt

```diff
@@ -6,7 +6,11 @@
 colorama
 cray-product-catalog
 PyYAML
+requests==2.25.1
```

Comment:
The `requests` package is already available in the virtual environment from the system Python packages. See later in this file.
```suggestion
```
Is there a reason you need this specific version of the package?